src/routes/channel.routes.js

```diff
@@ -24,3 +25,4 @@ router.post("/create/in/:id", [isAuthorized, isWorkspaceAdmin], createChannel);
 router.delete("/delete/:id", [isAuthorized, isChannelAdmin], deleteChannel);
 router.put("/update/:id", [isAuthorized, isChannelAdmin], updateChannel);
 router.post("/addUser/:id", [isAuthorized, isChannelAdmin], AddUserToChannel);
+router.get("/getUsers/:id", [isAuthorized, isChannelAdmin], getUsers);
```

Comment:
Add the isChannelMember instead of isChannelAdmin. As all users can see who is present in channel